api_volontaria/apps/position/models.py

```diff
@@ -0,0 +1,184 @@
+from datetime import datetime, timedelta
+import pytz
+from babel.dates import format_date
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_save, pre_delete
+from django.dispatch import receiver
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.auth import get_user_model
+from dry_rest_permissions.generics import authenticated_users
+# from api_volontaria.email import EmailAPI
+
+User = get_user_model()
+
+
+class Position(models.Model):
+    """
+    This class represents a position that an organization can advertise and that volunteer can apply to.
+    """
+
+    class Meta:
+        verbose_name = _("Position")
+        verbose_name_plural = _("Positions")
+
+    name = models.CharField(
+        verbose_name=_("Name"),
+        max_length=100,
+    )
+
+    description = models.TextField(
+        verbose_name="Description",
+    )
+
+    hourly_wage = models.FloatField(
+        verbose_name=_("Hourly wage"),
+    )
```

Comment:
Since you're in last little details, i think you should take a look on this package: [https://github.com/django-money/django-money#django-money](Django-Money)

It's easy to use, it will resolve the potential problem of currency, and it's the kind of package you will want to use in all your future project. 

You can just replace your FloatField with it.

Reply:
done

Reply:
@RignonNoel 

This change has triggered a bug when running test_update_position_as_an_admin in test_view_positions.py

Error Message is:
FAIL: test_update_position_as_admin (api_volontaria.apps.position.test_view_positions.PositionsTests)
Ensure we can update a position if we are an admin.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vardane/Documents/api_volontaria/API-Volontaria/api_volontaria/apps/position/test_view_positions.py", line 152, in test_update_position_as_admin
    self.assertEqual(
AssertionError: '16.50' != 16.5

When replacing 16.5 in content with float(16.5), error message becomes:

TypeError: Object of type MoneyField is not JSON serializable 

So I read Django-Money documentation about how to make a MoneyField serializable.

It seems to me it should be already serializable in our code since we state 
fields = '__all__' when defining the Position class in position/serializers.py

Unclear to me how and where code should be modified to make this MoneyField "hourly_wage" serializable.

Any suggestion about how to fix this?